app/services/account.py

```diff
@@ -19,7 +21,43 @@ async def create_account(dto: AccountCreate, db: Session):
         created_by="system",
         updated_by="system",
     )
+    new_account_response = AccountCreateResponse(
+        email=dto.email,
+        first_name=dto.first_name,
+        last_name=dto.last_name,
+        organization_id=None,
+    )
     db.add(new_account)
     db.commit()
-    db.refresh(new_account)
-    return new_account
+
+    return new_account_response
+
+
+async def account_signin(dto: AccountCreate, db: Session):
+    if not get_by_email(db, dto.email):
+        return {"error": "the account does not exist"}
+
+    hashed_pw = Hasher.get_password_hash(dto.password)
+    account = (
+        db.query(Account)
+        .filter(
+            Account.email == dto.email, Hasher.verify_password(dto.password, hashed_pw)
+        )
+        .first()
+    )
+    if account is None:
+        # raise HTTPException(status_code=401, detail="Invalid credentials")
+        return {"error": "wrong password"}
+
+    secret_key = "123456"
+    algo = "HS256"
+    payload = {
+        "sub": account.email,
+        "exp": time.time() + 7200,
```

Comment:
maybe change the valid time to sth longer like 8 hours? 2 hours seem a little too short.